pkg/apis/crd/v1alpha1/types.go

```diff
@@ -457,6 +464,14 @@ type NetworkPolicyPort struct {
 	EndPort *int32 `json:"endPort,omitempty"`
 }
 
+// ServiceReference represents a reference to a v1.Service.
+type ServiceReference struct {
+	// Name of the Service
+	Name string `json:"name,omitempty"`
+	// Namespace of the Service
```

Comment:
maybe add the requirements of these? Name is required. Namespace is required for ACNP usage

Reply:
ACNP and ANP both use this struct. I'm not sure how to do "Namespace is required for ACNP usage". But I have this requirement set in crds.yml.